### PR TITLE
fix: Correcting patch file and making changes to make jupyter work in dev env

### PR DESCRIPTION
**Pull request by Abdul-Muqadim-Arbisoft**
This PR addresses issues with the Tutor Jupyter plugin not functioning in development mode. Previously, the plugin lacked a necessary patch file for development environments. This update introduces a new patch file, **openedx-development-settings**, which ensures that the Jupyter plugin operates correctly in both development and local modes.

Additionally, this PR removes an incorrect reference to a non-existent patch file named **dev-docker-compose-services**. Confirmation that this patch file does not exist can be found in the official documentation here: [Tutor Patches Reference](https://docs.tutor.edly.io/reference/patches.html).

### CODE REVIEW

File: tutorjupyter/patches/local-docker-compose-dev-services [hunk 1]
```diff
@@ -0,0 +1,8 @@
+jupyterhub:
```
Comment (DawoudSheraz) [inline tutorjupyter/patches/local-docker-compose-dev-services:1]: Hmmm, why is this needed now? How was it working before?

Comment (DawoudSheraz) [reply tutorjupyter/patches/local-docker-compose-dev-services:1]: @Abdul-Muqadim-Arbisoft following up on this. Plus, having the info in the PR description will be helpful as well

Comment (Abdul-Muqadim-Arbisoft) [reply tutorjupyter/patches/local-docker-compose-dev-services:1]: Moving this PR to draft, since needs some more modifications, will add description as well , tutor Jupyter was never working before dev but it was only working on local, due to wrong patch file there is no such path as dev-docker-compose-services

**Review by DawoudSheraz — APPROVED** [review]
tested dev, working fine. 
nit: please add a changelog for this change.
